### PR TITLE
doc: archive old version documentation

### PR DESCRIPTION
**Pull request by leehom-liulihong**
# Summary

归档 github 版本文档，迁移声明中新增说明

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,74 +1,13 @@
...
-/>
-```
-
-IMPORTANT: The final image width and height will not necessarily match `downsampleWidth` and `downsampleHeight` but will just be a target for the per-platform logic on how to downsample.
-
-On Android, the logic for how to downsample is taken from [here](https://developer.android.com/topic/performance/graphics/load-bitmap). The image's aspect ratio will stay consistent after downsampling.
-
-On iOS, the max value of `downsampleWidth` and `downsampleHeight` will be used as the max pixel count for both dimensions in the final image. The image's aspect ratio will stay consistent after downsampling.
+- This repository has been migrated to Gitee: [OpenHarmony-SIG/rntpc_react-native-image-sequence](https://gitee.com/openharmony-sig/rntpc_react-native-image-sequence)
+- The package name has been changed to `@react-native-ohos/rntpc_react-native-image-sequence` and is now available for direct download from npm.
```
Comment (Louis-C7) [inline README.md]: @react-native-ohos/react-native-image-sequence

File: README.md [hunk 2]
```diff
@@ -1,74 +1,13 @@
-# react-native-image-sequence-2
-Based on [react-native-image-sequence](https://github.com/madsleejensen/react-native-image-sequence) with various bugs fixed.
+#  迁移声明
 
-native modules for handling image sequence animations. (created because i had performance issues with a javascript only solution like: https://github.com/remobile/react-native-image-animation)
+- 本仓库已迁移至 Gitee：[OpenHarmony-SIG/rntpc_react-native-image-sequence](https://gitee.com/openharmony-sig/rntpc_react-native-image-sequence)。
+- 包名已更改为 `@react-native-ohos/rntpc_react-native-image-sequence`，支持直接从 npm 下载。
```
Comment (Louis-C7) [inline README.md]: @react-native-ohos/react-native-image-sequence

**Review by Louis-C7 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (leehom-liulihong) [reply README.md]: 已修改

File: README.md (hunk 1 above)
Comment (leehom-liulihong) [reply README.md]: 已修改

File: README.md [hunk 3]
```diff
@@ -1,74 +1,13 @@
-# react-native-image-sequence-2
-Based on [react-native-image-sequence](https://github.com/madsleejensen/react-native-image-sequence) with various bugs fixed.
+#  迁移声明
 
-native modules for handling image sequence animations. (created because i had performance issues with a javascript only solution like: https://github.com/remobile/react-native-image-animation)
+- 本仓库已迁移至 Gitee：[OpenHarmony-SIG/rntpc_react-native-image-sequence](https://gitee.com/openharmony-sig/rntpc_react-native-image-sequence)。
+- 包名已更改为 `@react-native-oh-tpl/react-native-image-sequence-2`，支持直接从 npm 下载。
```
Comment (Louis-C7) [inline README.md]: @react-native-ohos/react-native-image-sequence

File: README.md [hunk 4]
```diff
@@ -1,74 +1,13 @@
...
-/>
-```
-
-IMPORTANT: The final image width and height will not necessarily match `downsampleWidth` and `downsampleHeight` but will just be a target for the per-platform logic on how to downsample.
-
-On Android, the logic for how to downsample is taken from [here](https://developer.android.com/topic/performance/graphics/load-bitmap). The image's aspect ratio will stay consistent after downsampling.
-
-On iOS, the max value of `downsampleWidth` and `downsampleHeight` will be used as the max pixel count for both dimensions in the final image. The image's aspect ratio will stay consistent after downsampling.
+- This repository has been migrated to Gitee: [OpenHarmony-SIG/rntpc_react-native-image-sequence](https://gitee.com/openharmony-sig/rntpc_react-native-image-sequence)
+- The package name has been changed to `@react-native-oh-tpl/react-native-image-sequence-2` and is now available for direct download from npm.
```
Comment (Louis-C7) [inline README.md]: 同上

**Review by Louis-C7 — CHANGES_REQUESTED** [review]

**Review by Louis-C7 — APPROVED** [review]

**Pull request merged**